srcs/vm/utils_2.c

```diff
@@ -0,0 +1,25 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   utils_2.c                                          :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: mvan-eng <mvan-eng@student.codam.nl>         +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2020/08/09 20:48:30 by mvan-eng      #+#    #+#                 */
+/*   Updated: 2020/08/13 15:25:05 by merlijn       ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "vm.h"
+
+/*
+**	IS_REGISTRY
+**	Checks if given argument is a valid registry
+*/
+
+int		is_registry(int arg)
+{
+	if (arg > 0 && arg <= 16)
+		return (1);
+	return (-1);
```

Comment:
Maybe we can import studbook.h and use 'true' and 'false' instead of 1 and -1

Reply:
I'm fine with 1 and -1 for Codam projects

Reply:
Als we dan al 1 en -1 gebruiken, dan lijkt me het volgende logischer:
1 = positief = true = OK
0 = neutral = false = NOT OK (E.G. in een functie die iets checked, als het niet klopt, is het niet perse een error)
-1 = negatief = error = ERROR